features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,18 @@
+Given("I visit the {string} page") do |string|
+    visit '/'
+end
+
+Given("The following user exist") do |table|
+    table.hashes.each do |user|
+        User.create!(user)
+    end
+end
+
+When("I click on {string}") do |string|
+    click_on(string)
```

Comment:
don't use `string` as a variable name, and be consistent with the with the brackets